doc/setup/install.rst

```diff
@@ -351,18 +356,23 @@ Installing dependencies
 
 Source installation requires manually installing dependencies:
 
-* Python 2.7.
+* Python 3.5+.
+
+  - on Linux and OS X, using your package manager if not installed by default
+
+    .. note:: on some system, ``python`` commands refers to Python 2 (outdated)
```

Comment:
Depending on your distribution,  ``python`` command (without a s) refers to Python 2...

Reply:
Isn't that what the note says?